### ISSUE TITLE
XML element for running specific actions

### ISSUE THREAD

**Issue by TravisMitchell** [CFD-GO/TCLB#278]
For multiphase methods, and certain temperature cases it could be useful to have an XML option of running various actions. E.g. use case is in thermocapillary flow where I want to run initialise all fields, solve the temperature update equation until steady state and then intialise the full simulation.

I view it would look something like:
```
<Init/>
<TempUpdate Iterations="1000000">
<StopChangeL2norm="1e-4"/>
>
<Solve Iterations="10000">
<VTK Iterations="100"/>
/>
```

**Comment by llaniewski** [comment]
I propose:
```xml
<Init/>
<Container> <!-- This keeps the Stop callback only for the RunAction -->
   <Stop L2ChangeNormBelow="1e-4" Times="10"/> <!-- Use standard Stop with custom Global -->
   <RunAction name="TempUpdate" Iterations="1000000"/> <!-- RunAction with name of action as argument, to keep with xml standard more -->
</Container>
<VTK Iterations="100"/>
<Solve Iterations="10000"/>
```

**Comment by llaniewski** [comment]
@TravisMitchell is it working right?

**Comment by TravisMitchell** [comment]
has been working nicely for me :). @ggruszczynski this can also take out the need for using the smoothing Additionals option in the multiphase models

**Comment by ggruszczynski** [comment]
Good idea!

**Comment by llaniewski** [comment]
fixed by #280
